rag/advanced_rag/harness/orchestrator/decompose.py

```
@@ -422,18 +542,88 @@ def _fallback_analysis(
+    existing_desc = {c.description.strip().lower() for c in ctx.claims}
+    pstate = {
+        "question": question,
+        "keywords": keywords,
+        "route": route,
+        "seed_chunks": tools.kbinfos.get("chunks", []),
+        "feedback": feedback,
+    }
+    _LOG.info("[Decompose] Level-2 replan (attempt %d) with feedback: %s", replan_count + 1, _snip(feedback))
+    try:
+        plan = await planner_node(pstate, tools)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)